css/properties/font-synthesis.json

```diff
@@ -48,6 +48,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "small-caps": {
+          "__compat": {
+            "description": "<code>small-caps</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "93"
+              },
+              "firefox_android": {
+                "version_added": "93"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": true
```

Comment:
Is this supposed to be marked as deprecated? The spec doesn't indicate anything about this being discouraged.

Reply:
```suggestion
              "deprecated": false
```

Reply:
No - I put it on the wrong key, which is how today is going, sorry!